pilot/cmd/pilot-agent/status/server.go

```diff
@@ -261,6 +261,7 @@ func (s *Server) Run(ctx context.Context) {
 	mux.HandleFunc(`/stats/prometheus`, s.handleStats)
 	mux.HandleFunc(quitPath, s.handleQuit)
 	mux.HandleFunc("/app-health/", s.handleAppProbe)
```

Comment:
Do we need/want a new path? Or just add a field to `Prober` (from KubeAppProbers) to pass the TCP settings?

Reply:
+1 this still belongs to the same app probe. tcp-probe be same level with /debug can caused confusion to think it's istio built-in. or at least `app-tcp-probe`